libraries/version.rb

```diff
@@ -186,10 +183,9 @@ def initialize
 
     private
 
-    # Win32API call to GetSystemMetrics(SM_SERVERR2)
     # returns: The build number if the system is Windows Server 2003 R2; otherwise, 0.
     def sm_serverr2
```

Comment:
We can probably just remove this method since we haven't support 2003 for a long time.

Reply:
removed